types/mui-datatables/index.d.ts

```diff
@@ -84,24 +83,24 @@ interface MUIDataTableTextLabelsViewColumns {
 }
 
 interface MUIDataTableTextLabelsSelectedRows {
-    text: string;
     delete: string;
     deleteAria: string;
+    text: string;
 }
 
 export interface MUIDataTableColumn {
-    name: string;
     label?: string;
+    name: string;
     options?: MUIDataTableColumnOptions;
 }
 
 export interface MUIDataTableTextLabels {
     body: MUIDataTableTextLabelsBody;
+    filter: MUIDataTableTextLabelsFilter;
     pagination: MUIDataTableTextLabelsPagination;
+    selectedRows: MUIDataTableTextLabelsSelectedRows;
     toolbar: MUIDataTableTextLabelsToolbar;
-    filter: MUIDataTableTextLabelsFilter;
```

Comment:
Why is this re-ordered?

Reply:
Alphabetized properties inside interface